android-connection/noto-fonts-4-android.xml

```diff
@@ -1804,4 +1804,27 @@
             NotoSerifOldUyghur-Regular.ttf
         </font>
 	</family>
+	<family lang="und-Cpmn">
+		<font weight="400" style="normal" path="unhinted/ttf/NotoSansCyproMinoan">
+            NotoSansCyproMinoan-Regular.ttf
+        </font>
+	</family>
+	<family lang="und-Tnsa">
+		<font weight="400" style="normal" fallbackFor="serif" postScriptName="NotoSansTangsa-Regular" path="unhinted/slim-variable-ttf">
+	          NotoSansTangsa-VF.ttf
+	          <axis tag="wght" stylevalue="400"/>
+	      </font>
+		<font weight="500" style="normal" fallbackFor="serif" postScriptName="NotoSansTangsa-Regular" path="unhinted/slim-variable-ttf">
+	          NotoSansTangsa-VF.ttf
+	          <axis tag="wght" stylevalue="500"/>
+	      </font>
+		<font weight="600" style="normal" fallbackFor="serif" postScriptName="NotoSansTangsa-Regular" path="unhinted/slim-variable-ttf">
+	          NotoSansTangsa-VF.ttf
+	          <axis tag="wght" stylevalue="600"/>
+	      </font>
+		<font weight="700" style="normal" fallbackFor="serif" postScriptName="NotoSansTangsa-Regular" path="unhinted/slim-variable-ttf">
+	          NotoSansTangsa-VF.ttf
+	          <axis tag="wght" stylevalue="700"/>
+	      </font>
+	</family>
 </familyset>
```

Comment:
since this is unrelated - I think? - to the noto_fonts_for_android_test.py change my suggestion would be to do it in separate PRs. That is, make another branch that's a copy of this one and revert changes to one of the files on each branch.

Reply:
will do (re branch)